Gems/Atom/RHI/Metal/Code/Source/RHI/BindlessArgumentBuffer.cpp

```diff
@@ -60,6 +56,14 @@ namespace AZ::Metal
             m_bindlessRWTextureArgBuffer->Init(device, argBufferDescriptors, "ArgumentBuffer_BindlessRWTextures");
             mtlArgBuffers.push_back(m_bindlessRWTextureArgBuffer->GetArgEncoderBuffer());
             mtlArgBufferOffsets.push_back(m_bindlessRWTextureArgBuffer->GetOffset());
+            
+            //Unbounded read cube textures
+            argDescriptor.access = MTLArgumentAccessReadOnly;
+            argDescriptor.textureType = MTLTextureTypeCube;
+            argBufferDescriptors[0] = argDescriptor;
+            m_bindlessRWTextureArgBuffer->Init(device, argBufferDescriptors, "ArgumentBuffer_BindlessRWTextures");
```

Comment:
This is same as line 57. 
It should be m_bindlessCubeTextureArgBuffer. The name also need to be updated